Source/WSideMenuVC.swift

```diff
@@ -196,6 +257,16 @@ public class WSideMenuVC: WSizeVC {
             closeSideMenu()
         }
     }
+    
+    public func hideStatusBar(hide: Bool) {
+        statusBarHidden = hide
+        
+        if (hide) {
```

Comment:
this could just be a one-liner `UIApplication.sharedApplication().delegate?.window!!.windowLevel = hide ? UIWindowLevelStatusBar : UIWindowLevelNormal`.

Also, are we force unwrapping `window` here?